addons/udes_delivery_control/views/stock_picking_vehicle_type.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_stock_picking_vehicle_type_list" model="ir.ui.view">
+        <field name="name">stock.picking.vehicle.type.list</field>
+        <field name="model">stock.picking.vehicle.type</field>
+        <field name="arch" type="xml">
+            <tree string="Attributes" editable="top">
+                <field name="sequence" widget="handle"/>
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- Menu Action -->
+    <record id="action_stock_picking_vehicle_type" model="ir.actions.act_window">
+            <field name="name">Vehicle Type</field>
```

Comment:
Nit: Can we change this to the plural `Vehicle Types`? Like we do for Operation Types in the same menu.